src/main/java/ch/ivyteam/ivy/maven/IarPackagingMojo.java

```diff
@@ -38,46 +38,50 @@
 
 /**
  * Packs the compiled project as ivy-archive (IAR).
- * 
+ *
  * @author Reguel Wermelinger
  * @since 6.0.0
  */
 @Mojo(name = IarPackagingMojo.GOAL)
 public class IarPackagingMojo extends AbstractMojo {
   public static final String GOAL = "pack-iar";
-  private static final String[] DEFAULT_INCLUDES = new String[] {"**/*"};
-  static final String[] DEFAULT_EXCLUDES = new String[] {"target", "%regex[target/(?!classes/).*]"};

```

Comment:
I've changed the approach to include to file-sets:
1. one for all things despite the 'target' dir
2. and another one for fine-selected 'target' contents -> 'classes' + 'src_hd'

... I think this is easier to manage than reg-ex.
Just share if you see negative effects coming @alexsuter 
https://github.com/axonivy/project-build-plugin/commit/b5e19776180b67f1d6c030840eae6aa9cc4166e1